Form/FormMapper.php

```diff
@@ -275,6 +280,21 @@ public function addHelp($name, $help)
     }
 
     /**
+     * Symfony default form class sadly can't handle
+     * form element with dots in its name (when data
+     * get bound, the default dataMapper is a PropertyPathMapper).
+     * So use this trick to avoid any issue.
+     *
+     * @param string $fieldName
+     *
+     * @return string
+     */
+    public function sanitizeFieldName($fieldName)
```

Comment:
Yes I can, I hoped to use this methos here https://github.com/sonata-project/SonataAdminBundle/pull/4405/files#diff-c770a6f14ff46a6b7db321023b50709eR34 but I can